archive/zip_archiver.go

```diff
@@ -58,6 +59,8 @@ func (a *ZipArchiver) ArchiveFile(infilename string) error {
 	}
 	fh.Name = fi.Name()
 	fh.Method = zip.Deflate
+	var mtime time.Time
+	fh.SetModTime(mtime)
```

Comment:
Per [the docs](https://golang.org/pkg/archive/zip/#FileHeader.SetModTime) should just set `Modified` directly:
```golang
fh.Modified = time.Time{}
```

Reply:
`FileInfoHeader` uses `SetModTime()` which sets `Modified`, `ModifiedTime`, `ModifiedDate`. Updating only `Modified` would still result in changing checksum.

Reply:
No, the [`CreateHeader`](https://golang.org/src/archive/zip/writer.go?s=8093:8157#L304) method overwrites those other deprecated fields as well, it only reads `Modified`.

Reply:
Nevermind, you are correct, I see in the internal writing that it does actually write those out :( so much for their deprecation.